kodilla-testing/src/main/java/com/kodilla/testing/shape/Circle.java

```diff
@@ -35,4 +36,10 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(name, radius);
     }
+
+    @Override
+    public String toString() {
+        DecimalFormat df = new DecimalFormat("0.00");
+        return this.name + ", field = " + df.format(getField());
```

Comment:
Ja bym dał tylko name i promień, np. Circle(5), po co dawać pole? Jest to wartość wyliczana